src/main/java/neqsim/thermo/phase/PhaseWax.java

```diff
@@ -46,10 +46,9 @@ public void init(double totalNumberOfMoles, int numberOfComponents, int type, in
 
   /** {@inheritDoc} */
   @Override
-  public void addcomponent(String componentName, double molesInPhase, double moles,
-      int compNumber) {
-    super.addcomponent(molesInPhase);
-    componentArray[compNumber] = new ComponentWax(componentName, moles, molesInPhase, compNumber);
+  public void addcomponent(String name, double moles, double molesInPhase, int compNumber) {
+    super.addcomponent(name, moles);
```

Comment:
should be addcomponent(name, molesInPhase?)

Reply:
I did a check. You are right. It should be addcomponent(name, molesInPhase)